lua/packer/result.lua

```diff
@@ -1,4 +1,5 @@
--- A simple Result<V, E> type to simplify control flow with installers and updaters
+--- A simple Result<V, E> type to simplify control flow with installers and updaters
```

Comment:
Is this a change to make a documentation comment? Not sure what this change is otherwise.

Reply:
@wbthomason I presume so since the type won't appear in lsp hover docs etc if it isn't a doc comment using `---` I've been meaning to suggest using more of this pattern + emmy annotations since once sumneko is setup right it's really helpful to get to know parts of a code base or just remember how things work

Reply:
Actually correction I just checked and sumneko will pick it up, I think it's function annotation `--@param` won't work as an emmy lua annotation only `---@param` so assumed the same was true here but isn't

Reply:
My first thought was to make it more Emmy friendly since I had to figure out on my own how this worked, but since I'm still a newbie in Lua (and contributing in general) I thought it was better leave it as it is and maybe document better in another PR